scripts/src/wranglerPagesDev.ts

```diff
@@ -43,12 +43,27 @@ export const wranglerPagesDev = async ({
 		},
 	});
 
-	wranglerProcess.stdout?.on("data", (chunk) => {
-		logger.debug(chunk.toString());
+	wranglerProcess.stdout.on("readable", () => {
+		let chunk: any;
+		while (null !== (chunk = wranglerProcess.stdout.read())) {
+			logger.debug(chunk.toString());
+		}
 	});
 
-	wranglerProcess.stderr?.on("data", (chunk) => {
-		logger.debug(chunk.toString());
+	wranglerProcess.stderr.on("readable", () => {
+		let chunk: any;
+		while (null !== (chunk = wranglerProcess.stderr.read())) {
+			const chunkStr: string = chunk.toString();
```

Comment:
Probably don't need the explicit type?

```suggestion
			const chunkStr = chunk.toString();
```